### PR TITLE
feat(component): responsive tabs

### PR DESCRIPTION
**Pull request by gepifanio**
# Description

Fixes: https://app.zenhub.com/workspaces/study-5ab1c1515aba7b1bed1674d1/issues/unimelb/find-a-course/1491

- Responsive tabs for both tab types
- Has a 1 second delay to give the children components time to load and then grabs the width of them

https://deploy-preview-1376--pattern-lib-unimelb.netlify.com/?selectedKind=Tabs&selectedStory=Tabs%20%28alt%29&full=0&addons=1&stories=1&panelRight=0&addonPanel=REACT_STORYBOOK%2Freadme%2Fpanel

https://deploy-preview-1376--pattern-lib-unimelb.netlify.com/?selectedKind=Tabs&selectedStory=Tabs%2C%20isolated&full=0&addons=1&stories=1&panelRight=0&addonPanel=REACT_STORYBOOK%2Freadme%2Fpanel

## Type of change

Please delete options that are not relevant.

- [x] New feature (non-breaking change which adds functionality)

# Checklist:

- [x] Make sure to do not repeat yourself
- [x] Crossbrowser testing (IE11, Safari 8+, iOS 8.4+, Android 4.4+, Firefox ESR (v52.x), iPhone (4s,6), iPad 2, Galaxy s5) 
- [x] My changes generate no new warnings
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Comment by gepifanio** [comment]
Comment: Deploy Started

Building with commit bfa9f89137dcbd3886ea2cb7c851dfe36085e30a

https://app.netlify.com/sites/pattern-lib-unimelb/deploys/5df071a04dfd5c00081d6557

File: components/tabs/Tabs.vue [hunk 1]
```diff
@@ -138,8 +176,50 @@ export default {
...
+    // Hack to get child components to properly load.
+    setTimeout(() => {
+      this.tabsWidth = this.calculateTabsWidth();
+      this.showControls = this.hasControls();
+
+      window.addEventListener('resize', this.checkControls);
+    }, 2000);
   },
   methods: {
+    rightClick() {
```
Comment (gepifanio) [inline components/tabs/Tabs.vue]: following the pattern, might be better change to `nextTab` 
`prevTab`

**Comment by gepifanio** [comment]
@TimMansell looks awesome ! 🎉

**Comment by TimMansell** [comment]
> Looking good. As discussed the soft border highlight inherited from the old design can be removed (from the new design).

As discussed this is an a11y feature (across all components) so we will keep it as is

File: components/tabs/Tabs.vue (hunk 1 above)
Comment (TimMansell) [reply components/tabs/Tabs.vue]: I have changed this now.

**Pull request merged**